src/app/public/modules/auth-http-client/auth-interceptor.ts

```diff
@@ -52,11 +53,14 @@ function removeSkyParams(request: HttpRequest<any>): HttpRequest<any> {
   return request;
 }
 
+export const DEFAULT_PERMISSION_SCOPE = new InjectionToken<string>('default_permission_scope');
```

Comment:
I'd prefer that all files have only one export to facilitate Single Responsibility, see: https://angular.io/guide/styleguide#single-responsibility

Reply:
Moved to it's own file.